common/supplemental/supplementalData.xml

```diff
@@ -4787,91 +4787,91 @@ XXX Code for transations where no currency is involved
-              <era type="0" start="622-7-15" code="islamic-tbla" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="islamic-umalqura">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15" code="islamic-umalqura" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="chinese">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2636-01-01" code="chinese"/> <!-- 'sequential' year -->
+              <era type="0" start="-2636-01-01"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="hebrew">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-3760-10-7" code="hebrew" aliases="am"/>
+              <era type="0" start="-3760-10-7" code="am"/> <!-- Anno Mundi -->
           </eras>
         </calendar>
         <calendar type="buddhist">
           <calendarSystem type="solar" />
           <eras>
-              <era type="0" start="-542-01-01" code="buddhist" aliases="be"/>
+              <era type="0" start="-542-01-01" code="be"/> <!-- Buddhist Era -->
           </eras>
         </calendar>
         <calendar type="coptic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="284-08-28" code="coptic-inverse"/>
-              <era type="1" start="284-08-29" code="coptic"/>
+              <era type="0" end="284-08-28" code="bd"/> <!-- Before Diocletian. NOTE: No evidence this era is used -->
+              <era type="1" start="284-08-29" code="am" primary="true"/> <!-- Anno Martyrum -->
           </eras>
         </calendar>
         <calendar type="persian">
           <calendarSystem type="solar"/>
           <eras>
-              <era type="0" start="622-01-01" code="persian" aliases="ap"/>
+              <era type="0" start="622-01-01" code="ap" aliases="sh hs"/> <!-- Anno Persico -->
           </eras>
         </calendar>
         <calendar type="dangi">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2332-01-01" code="dangi"/> <!-- 'sequential' year -->
+              <era type="0" start="-2332-01-01"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="ethiopic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="8-08-28" code="ethioaa" aliases="ethiopic-amete-alem mundi"/>
-              <era type="1" start="8-08-29" code="ethiopic" aliases="incar"/>
+              <era type="0" end="8-08-28" code="aa" aliases="mundi"/> <!-- Amete Alem -->
+              <era type="1" start="8-08-29" code="am" aliases="incar" primary="true"/> <!-- Amete Mihret -->
```

Comment:
I think the Amete Alem era should be primary, because then both ethiopic calendars have the same extended year.

Reply:
Hmm interesting thought

Reply:
https://github.com/tc39/proposal-intl-era-monthcode/issues/30